### ISSUE TITLE
Coordinate of the rectangle

### ISSUE THREAD

**Issue by Alanhamel** [boyw165/algorithm-challenge#1]
Dear Boy Wang,

I would like to know if you could implement the coordinate of the rectangle,
for example if you have a rectangle of 12, could it be possible to print the first line and first column and the height and width of the biggest rectangle to know exactly the position of the rectangle in the matrix.

Thank you

**Comment by tcw165** [comment]
Hi Alan,

What do you mean by "a rectangle of 12"? I don't quite understand your question, what code in my repo your question refers to?

**Comment by tcw165** [comment]
Oh, is this one, https://github.com/boyw165/algorithm-challenge-python/blob/master/maximal-rectangle/README.md ?

**Comment by tcw165** [comment]
I don't have the code by hand, but I what you asked is possible.

**Comment by Alanhamel** [comment]
Hello Boy,

Yes the issue was for the repo called maximal-rectangle. 
Could it be possible for you to implement this featured ?

Thank you very much.

**Comment by tcw165** [comment]
Yes, I'm working on this. Honestly, it might take some work because I am kinda confused by my previous approach.

**Comment by tcw165** [comment]
Hey, I'm still working on this but you could follow my latest blog post to find out the rectangle with x, y, w and h.
https://medium.com/@boyw165/algorithm-largest-area-in-histogram-84cc70500f0c

**Comment by Alanhamel** [comment]
Thank you i will take a look.
